ArcGISRuntimeSDKQt_CppSamples/Maps/DownloadPreplannedMap/DownloadPreplannedMap.cpp

```diff
@@ -217,10 +183,61 @@ void DownloadPreplannedMap::loadPreplannedMapAreas()
         return;
       }
 
-      Graphic* areaOfInterest = new Graphic(mapArea->areaOfInterest(), m_graphicParent.get());
+      Graphic* areaOfInterest = new Graphic(mapArea->areaOfInterest(), this);
       m_graphicsOverlay->graphics()->append(areaOfInterest);
     });
 
     mapArea->load();
   }
 }
+
+void DownloadPreplannedMap::downloadPreplannedMapArea(const QString& path, int index)
+{
+  connect(m_offlineMapTask, &OfflineMapTask::createDefaultDownloadPreplannedOfflineMapParametersCompleted, this, [this, path] (QUuid ,const DownloadPreplannedOfflineMapParameters& parameters)
```

Comment:
if this method get's called multiple times, you will end up with multiple connections for this. Might be better to move the connection to the point where you first create the `m_offlineMapTask` so that it will be setup once only